docs/sphinx/index.rst

```diff
@@ -101,6 +106,7 @@ Uberenv has a few options that allow you to control how dependencies are built:
   ``--install``           Fully install target, not just dependencies    **False**
   ``--run_tests``         Invoke tests during build and against install  **False**
   ``--project-json``      File for project specific settings             See :ref:`project_configuration`
+  ``--triplet``           (vcpkg) Target architecture and linkage        ``x86-Windows``
```

Comment:
> In the code, the default appears to be ( from_environment() || "x86-Windows").

Are you suggesting that the default should read `from_environment() || "x86-Windows"` ?

In other words, what would you suggest to replace `x86-Windows` ?

Reply:
I was just noticing that the _documentation_ does not describe the default value accurately, since by default (meaning "if not specified on command line") uberenv will use the env variable, and only if not defined use "x86-Windows".
Sorry for the confusion.

Reply:
In the end I think we are saying the same thing. So, to answer your question: yes I think the documentation should describe what the code does, with something _like_ `from_environment() || "x86-Windows"`.

Reply:
Thanks @adrienbernede  -- does this work? #41